### DISCUSSION TITLE
Plan options.

### DISCUSSION THREAD

**Discussion by JeJaDiaz** [planetscale/discussion#560] [Q&A] [answered]
Can i down grade from Scaler Pro to Hobby free?

**Comment by jgreet** [comment, marked as answer]
We can downgrade a database from a Scaler or Scaler Pro plan to Hobby for you if the database usage fits within the limits of the free plan. If that's something you're interested in, open a support ticket to get started:
https://support.planetscale.com/hc/en-us/requests/new

**Comment by abdulazizahwan** [comment]
Which countries is the PlanetScale hobby tier available in? I don't find Indonesia
